app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -1142,7 +1142,7 @@ describe("FormAudience", () => {
     });
 
     expect(
-      within(screen.queryByTestId("audience-values") as HTMLElement)
+      within(screen.queryByTestId("population-percent-top-row") as HTMLElement)
```

Comment:
@eliserichards This was using a different test id than all the queries above.  Changing it to match fixed the test, does this look right to you?

Reply:
yup yup I remembered this

Reply:
yes, that's correct 👍